geode-core/src/main/java/org/apache/geode/cache/client/internal/ExecuteRegionFunctionSingleHopOp.java

```diff
@@ -297,13 +297,14 @@ public ExecuteRegionFunctionSingleHopOpImpl(String region, String functionId,
     }
 
     private void addBytes(byte functionState) {
-      if (ConnectionImpl
-          .getClientFunctionTimeout() == ConnectionImpl.DEFAULT_CLIENT_FUNCTION_TIMEOUT) {
+      ConnectionImpl conn = ConnectionImpl.getAnyInstance();
```

Comment:
Ok, this smells bad. Sorry to keep asking you to change directions on this. Really the first attempt, using `final static`, was probably the cleanest without major refactor to move the timeout to where it belongs. Having to just grab any connection just makes this dirtier.